src/main/webapp/components/BuildSnapshotContainer.js

```diff
@@ -9,7 +9,15 @@ import {BuildSnapshot} from "./BuildSnapshot";
  */
 export const BuildSnapshotContainer = React.memo((props) =>
   {
-    const SOURCE = '/builders';
+    // Number of entries to be pulled.
+    const NUMBER = 2;
+
+    // The starting point from which the entries are selected.
+    // Relative to the starting index.
+    const OFFSET = 0;
```

Comment:
This makes sense as the initial offset. Just to clarify: when the user goes to the next page, I'd expect the UI to query for the next NUMBER entries starting at offset (NUMBER+1). How would this work?

Reply:
I believe this is outside of this PR since it may require some addition React State logic to achieve.

The logic will probably be as follows:
If the user goes to page 2, the NUMBER param will not change, but OFFSET= will increase to be a multiple of NUMBER (e.g. offset for page two is NUMBER * 1). The formula would be **OFFSET=NUMBER * PAGENUMBER-1**

Reply:
I'll express OFFSET as NUMBER * 0 for the time being.

Reply:
Note that OFFSET right now is a const, so this doesn't allow to dynamically change it AFAIK.
Probably we'll need to turn it into React state later.